Visualize application rewrite - OSD PR3 (#12324)
opensearch-project/documentation-website: _dashboards/visualize/one-dimensional-data.md

   The visualization pane displays a gauge displaying a count of the documents in the index pattern. For the `opensearch_dashboards_sample_data_flights` data, this is `13059` if the date range contains all the documents.
   {: .note}

1. Select the **Data** tab.

1. In the **Metrics** box in the **Data** tab, select {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/arrow-down-icon.png" class="inline-icon" alt="expand icon"/>{:/} (expand) **Metric count** to expand the **Metrics** list.

1. In the **Aggregation** box, select the aggregation you want to display.

   For this tutorial, select `Median`.

1. In the **Field** box, select the field you want to display.

   For this tutorial, select `FlightTimeMin`.

1. Select **Update** in the lower right of the visualization tool to view the gauge in the display panel.

   The gauge displays a value `502.775`, presented as the full range of gauge.

   If your visualization displays a different value, make sure that your [time filter]({{site.url}}{{site.baseurl}}/dashboards/discover/time-filter/) window is large enough to encompass all the sample flight data.
   {: .note}

> 1. Now that you have some idea of the range of the data, change the ranges so that the displayed value.

Reviewer: This sentence is truncated mid-thought ("…so that the displayed value."). Please complete it — something like *"…so that the displayed value falls in the middle range."*

Author: ```suggestion
1. Now that you have some idea of the magnitude of the data, change the display range so that the displayed value falls within it.
```

   1. Select the **Options** tab.

   1. In the **Ranges** panel, edit the three default ranges as shown in the following table:

      ≥    0 | < 250
      ≥ 250 | < 500
      ≥ 500 | < 750

   1. Select **Update**.

1. To compare two different conditions in one visualization, bucket the data.

   1. In the **Buckets** pane, select **Add**.

   1. In the **Add bucket** drop-down, select **Split group**.

   1. In the **Aggregation** drop-down, select an aggregation type for the rows.

      For this example, select **Terms**.

   1. In the Field drop-down, select a field for the row buckets. For this example, select the boolean value **FlightDelay**.

   1. Select **Update**.

   The visualization displays two gauges, with median flight times for delayed and undelayed flights, as shown in the following image.

   <img src="{{site.url}}{{site.baseurl}}/images/dashboards/example-gauge-flight-time.png" alt="Visualize application" width="60%">

1. Save the visualization as described in [Saving a data visualization]({{site.url}}{{site.baseurl}}/dashboards/visualize/saving-a-viz/).

## Building a goal visualization

> The  Goal visualization displays a value in a speedometer-like gauge. The shows a the value as a proportion of a predefined goal. The visualization also displays the metric, either as an absolute value or as a percentage of the goal.

Reviewer: Mangled phrasing ("The shows a the value"). Suggested rewrite:

```suggestion
The Goal visualization displays a value in a speedometer-like gauge. It shows the value as a proportion of a predefined goal. The visualization also displays the metric, either as an absolute value or as a percentage of the goal.
```

1. In the **Aggregation** box, select the aggregation you want to display.

   For this tutorial, select `Sum`.

1. In the **Field** box, select the field you want to display.

   For this tutorial, select `DistanceKilometers`.

1. To see how close the dataset is to recording a total of a hundred million kilometers, set 100,000,000 as the goal.

   1. Select the **Options** tab.

   1. In the **Ranges** boxes, enter `1e8` as the top of the range.

      Numerical entries cannot take group separators. It's easier to use exponential notation for large numbers.
      {: .note}

   1. De-select **Percentage mode**.

   1. Select **Update**.

   The visualization displays `92,616,288.34`, the sum of all flight distances in kilometers. This value shows as about 93% of the gauge maximum, which is calibrated to the goal, as shown in the following image.

   <img src="{{site.url}}{{site.baseurl}}/images/dashboards/example-goal-total-km.png" alt="Visualize application" width="60%">

>    1. (Optional) Combine Gauge and Goal displays by adding a scale as described in [Building a guage visualization](#building-a-gauge-visualization).

Reviewer: Typo: `guage` → `gauge`.

```suggestion
   1. (Optional) Combine Gauge and Goal displays by adding a scale as described in [Building a gauge visualization](#building-a-gauge-visualization).
```

   {: .note}

   If your visualization displays a different value, make sure that your [time filter]({{site.url}}{{site.baseurl}}/dashboards/discover/time-filter/) window is large enough to encompass all the sample flight data.
   {: .note}

1. In the **Metrics** box in the **Data** tab, select {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/arrow-down-icon.png" class="inline-icon" alt="expand icon"/>{:/} (expand) **Slice size count** to expand the **Metrics** list.

1. In the **Aggregation** box, select the aggregation you want to display.

   For a pie chart, bucketed values of the metric must add to 100%. Only those aggregations with this property are displayed.

   For this tutorial, select `Unique Count`.

1. In the **Field** box, select the field you want to display.

   For this tutorial, select `FlightNum`.

1. Select **Update** in the lower right of the visualization tool to view the pie chart in the display panel.

>    The gauge displays a value `12,932`, presented again as a single-value annular pie chart.

Reviewer: Factual error — this is the pie-chart procedure, not gauge. "The gauge displays a value" should read "The pie chart displays a value".

```suggestion
   The pie chart displays a value `12,932`, presented again as a single-value annular pie chart.
```
